File: paas-common-utils/src/main/java/com/zzstack/paas/underlying/utils/bean/ResultBean.java
package com.zzstack.paas.underlying.utils.bean;

import com.zzstack.paas.underlying.utils.consts.CONSTS;

public class ResultBean {
	
	private int retCode;
	private String retInfo;
	
	public ResultBean() {
		this.retCode = CONSTS.REVOKE_OK;
		this.retInfo = "";
	}

	public int getRetCode() {
		return retCode;
	}

	public void setRetCode(int retCode) {
		this.retCode = retCode;
	}

	public String getRetInfo() {
		return retInfo;
	}

	public void setRetInfo(String retInfo) {
		this.retInfo = retInfo;
	}
	
	@Override
	public String toString() {
		return "ResultBean [RET_CODE=" + retCode + ", RET_INFO=" + retInfo + "]";
	}
	
}

File: paas-metasvr-go/pkg/meta/proto/result_bean.go
package proto

import "github.com/maoge/paas-metasvr-go/pkg/consts"

type ResultBean struct {
	RET_CODE int
	RET_INFO string
}

func NewResultBean() *ResultBean {
	return &ResultBean{
		RET_CODE: consts.REVOKE_OK,
		RET_INFO: "",
	}
}
